Major Description Changes

gonzalez-mark opened the pull request:
I don't think I did this right

gonzalez-mark committed:
Create committee.html

gonzalez-mark committed:
Revert "Create committee.html"

This reverts commit 7f562c343d531d5b35f65a751c56502c0b8b4122.

gonzalez-mark committed:
Create committee.html

gonzalez-mark committed:
Major Description Changes

I dont know if i did the PR right

KhanWhale commented on index.html:
```diff
@@ -1499,7 +1499,7 @@ <h3>Esther Jung
                         <figcaption class="caption text-center">
                             <h3>Jaimie Lee
-                                <small class="major">Public Health, Psychology, and Public Policy</small>
+                                <small class="major">Public Health, Psychology, & Public Policy</small>
```
Hm, I think there's a typo here (should just be a ",")

gonzalez-mark replied:
Since Jaimie has 3 majors listed, the structure is as follows: "major, major, & major".

KhanWhale replied:
Ah, I see.

KhanWhale commented on index.html:
```diff
@@ -1382,7 +1382,7 @@ <h3>Marie Bellevue
                             <h3>Aniruddh Khanwale
                                 <!-- <small>loves making google forms</small> -->
-                                <small class="major">Materials Science & Engineering, EECS</small>
+                                <small class="major">Materials Science and Engineering, EECS</small>
```
There's nothing wrong here, but do you mind adding Energy Engineering at the end :O

KhanWhale commented on index.html:
```diff
@@ -1793,22 +1793,22 @@ <h3>Matthew Clark</h3>
         <h3>Shantanu Kadam</h3>
         <div class="line"></div>
-        Hey there! I'm a physics major who also appreciates math (I know you didn't see that coming) and cares about the environment. When I'm not at the chalkboard, you can find me swing dancing, cooking, daydreaming about Siberian Huskies, exploring Berkeley for vegan treats, perpetually drinking tea, reading, and (most importantly!) sleeping. Home sweet home is back in Alabama, so if you have  ...
+        Hey there! I'm a physics major who also appreciates math (I know you didn't see that coming) and cares about the environment. When I'm not at the chalkboard, you can find me swing dancing, cooking, daydreaming about Siberian Huskies, exploring Berkeley for vegan treats, perpetually drinking tea, reading, and (most importantly!) sleeping. Home sweet home is back in Alabama, so if you have  ...
```
This doesn't look like it's your fault, but see if you can Shantanu's bio to be formatted like everyone else's (i.e. multiple lines). Same thing if there's any other bios that for some reason are only one line.

KhanWhale reviewed (changes requested):
Do you mind removing committee.html **from the repo only?** Since this is still the "original" website, we don't have separate html files (yet). I don't want you to delete the file, because it will be useful on the redesign branch, just move it out of your "rohm-website" folder and make a new commit, also addressing the changes I mentioned. Overall though, looks really good, thank you!

gonzalez-mark committed:
Revert "Create committee.html"

This reverts commit 110bfb345a1a5cd775f95fc0c88588957ddb8f0e.

gonzalez-mark committed:
Commitee Description Update 2

Not sure if Jaimie's major needed changes. 3 majors are structured "major, major, & major".

gonzalez-mark committed:
Update index.html

KhanWhale reviewed (approved):
Looks good. Great job!
